scripts/install-zowe.sh

```diff
@@ -377,6 +395,36 @@ if [[ "$CI_IS_SMPE" = "yes" ]]; then
   echo "[${SCRIPT_NAME}] Zowe configuration is done, start installing xmem server ..."
   cd ${CIZT_SMPE_PATH_PREFIX}${CIZT_SMPE_PATH_DEFAULT}/xmem-server
   ${CIZT_INSTALL_DIR}/install-xmem-server.sh
+
+  # Copy xmem server PROCLIB, PARMLIB and LOADLIB members into targets
+  echo "calling zowe-install-xmem.sh with"
+  echo "  datasetprefix ${DATA_SET_PREFIX}"
+  echo "  zss loadlib   ${CIZT_ZSS_LOADLIB_DS_NAME}"
+  echo "  zss parmlib   ${CIZT_ZSS_PARMLIB_DS_NAME}"
+  echo "  zss proclib   ${CIZT_ZSS_PROCLIB_DS_NAME}"
+  cd $CIZT_ZOWE_ROOT_DIR/scripts/utils
+  chmod a+x ./zowe-install-xmem.sh 
```

Comment:
Please remove this line - if the script doesn't have execute permission, we need to know about it as a install defect